openshift/kfdef/kfctl_openshift_v1.5.0_odh.yaml

```diff
@@ -0,0 +1,83 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: kubeflow
+  namespace: kubeflow
+spec:
+  applications:
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/openshift/openshift-scc
+    name: openshift-scc
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/istio-1-1-1-Openshift
+    name: istio
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/cert-manager-1.5
+    name: cert-manager
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/kfp-tekton
+    name: kfp-tekton
+#Uncomment if you need KFP on Argo
+#Make sure to comment kfp-tekton
+#  - kustomizeConfig:
+#      repoRef:
+#        name: manifests
+#        path: openshift/openshiftstack/application/pipeline-agnostic
+#    name: pipeline
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/knative
+    name: knative
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/jupyter-web-app
+    name: jupyter-web-app
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/notebook-controller
+    name: notebook-controller
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/volumes-web-app
+    name: volumes-web-app
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/training-operator
+    name: training-operator
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/katib
+    name: katib
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/mpi-job
+    name: mpi-job
+  - kustomizeConfig:
+      repoRef:
+        name: manifests
+        path: openshift/openshiftstack/application/kfserving
```

Comment:
for IKS, we move to KServe. will you do the same change?

Reply:
Apologies for the delay. @VaishnaviHire Is finishing up the work to include kserve as part of this deployment